mtest/libmscore/albums/tst_albums.cpp

```diff
@@ -0,0 +1,162 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 MuseScore BVBA
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include <QtTest/QtTest>
+#include "mtest/testutils.h"
+#include "mscore/musescore.h"
+#include "mscore/preferences.h"
+#include "libmscore/measure.h"
+
+#define DIR QString("libmscore/albums/")
+
+using namespace Ms;
+
+//---------------------------------------------------------
+//   TestAlbums
+//---------------------------------------------------------
+
+class TestAlbums : public QObject, public MTest
+{
+    Q_OBJECT
+
+    void albumAddScore();
+    void albumItemDuration();
+    void albumItemEnable();
+    void albumItemBreaks();
+
+private slots:
+    void initTestCase();
+
+    void albumAddScoreTest() { albumAddScore(); }
+    void albumItemDurationTest() { albumItemDuration(); }
+    void albumItemEnableTest() { albumItemEnable(); }
+    void albumItemBreaksTest() { albumItemBreaks(); }
+};
+
+#define private public
```

Comment:
Huh??

Reply:
Allows me to test private members of the class.

Reply:
And disabling that didn't help.

Reply:
This was just a hack to get mtests to compile locally. It was not supposed to be part of the pull request. For some reason, qtquick_compiler_add_resources is not doing the job, so this forces it to fall back to using QT5_ADD_RESOURCES instead. Interestingly enough, however, qtquick_compiler_add_resources seems to work just fine when called from mscore/CMakeLists.txt.

Reply:
Yes, of course. I don't plan to include this in the completed pull request. :^)